<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-R8LNRUV" ignored="2" name="Test" tests="6" failures="1" timestamp="2022-10-20T17:07:33 SGT" time="363.609" errors="0">
  <testcase name="test_verify_title" time="0.543" classname="testCases.TestCase_SampleTest">
    <skipped/>
  </testcase> <!-- test_verify_title -->
  <testcase name="test_education_loan_standard_repayment_calculator_scn002" time="20.732" classname="testCases.TestCase_Scenario_0002"/>
  <testcase name="test_verify_title" time="0.035" classname="testCases.TestCase_SampleTest">
    <failure type="java.lang.AssertionError" message="expected [HomePage] but found [Personal Banking | Savings, Credit Cards and Loans | CIMB SG]">
      <![CDATA[java.lang.AssertionError: expected [HomePage] but found [Personal Banking | Savings, Credit Cards and Loans | CIMB SG]
at testCases.TestCase_SampleTest.test_verify_title(TestCase_SampleTest.java:20)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- test_verify_title -->
  <testcase name="test_education_loan_standard_repayment_calculator_scn002" time="16.394" classname="testCases.TestCase_Scenario_0002"/>
  <testcase name="test_education_loan_standard_repayment_calculator_scn002" time="10.392" classname="testCases.TestCase_Scenario_0002"/>
  <testcase name="test_education_loan_standard_repayment_calculator_scn002" time="10.003" classname="testCases.TestCase_Scenario_0002"/>
  <testcase name="@BeforeTest start_docker_grid" time="0.0" classname="testBase.TestBase">
    <ignored/>
  </testcase> <!-- @BeforeTest start_docker_grid -->
  <testcase name="@AfterTest stop_docker_grid" time="0.0" classname="testBase.TestBase">
    <ignored/>
  </testcase> <!-- @AfterTest stop_docker_grid -->
</testsuite> <!-- Test -->
